### PR TITLE
fix: Add 'cellHighlight' to grid styles cache key

### PR DESCRIPTION
**Pull request by bdow**

### CODE REVIEW

File: src/components/Table/styles.tsx [hunk 1]
```diff
@@ -54,8 +54,8 @@ export interface GridStyleDef {
 function memoizedTableStyles() {
   const cache: Record<string, GridStyle> = {};
   return (props?: GridStyleDef) => {
-    const { inlineEditing = false, grouped = false, rowHeight = "flexible", cellHighlight } = props || {};
-    const key = `${inlineEditing}|${grouped}|${rowHeight}`;
+    const { inlineEditing = false, grouped = false, rowHeight = "flexible", cellHighlight = false } = props || {};
+    const key = `${inlineEditing}|${grouped}|${rowHeight}|${cellHighlight}`;
```
Comment (stephenh) [inline src/components/Table/styles.tsx:58]: @bdow wdyt about a `JSON.stringify(props)` so that new props going forward are handled for free/

**Review by chr1sjf0x — APPROVED** [review]
TY 🍣

File: src/components/Table/styles.tsx (hunk 1 above)
Comment (bdow) [reply src/components/Table/styles.tsx:58]: I like the idea. I wonder how that'd look... Will try.

Comment (bdow) [reply src/components/Table/styles.tsx:58]: Would be worried props may not always be returned in the same order, so could be tricky. Might be able to do something like `Object.keys`, sort, get values, then stringify.

Comment (bdow) [reply src/components/Table/styles.tsx]: Pushed up a change, let me know what you think! Thanks for the suggestion.

File: src/components/Table/styles.tsx [hunk 2]
```diff
@@ -53,9 +54,12 @@ export interface GridStyleDef {
 
 function memoizedTableStyles() {
   const cache: Record<string, GridStyle> = {};
-  return (props?: GridStyleDef) => {
-    const { inlineEditing = false, grouped = false, rowHeight = "flexible", cellHighlight } = props || {};
-    const key = `${inlineEditing}|${grouped}|${rowHeight}`;
+  return (props: GridStyleDef = {}) => {
+    const { inlineEditing = false, grouped = false, rowHeight = "flexible", cellHighlight = false } = props;
```
Comment (stephenh) [inline src/components/Table/styles.tsx:58]: Nit: could drop this line now? Good call on the sort!

**Review by stephenh — APPROVED** [review]

File: src/components/Table/styles.tsx (hunk 2 above)
Comment (bdow) [reply src/components/Table/styles.tsx:58]: I don't think we can drop this line, we still use the destructured props to apply the styles.

Comment (stephenh) [reply src/components/Table/styles.tsx:58]: Oh right :-) Nm!

**Pull request merged**